components/fonts/hack-ttf/Makefile

```diff
@@ -0,0 +1,64 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Tim Mooney <Timothy.V.Mooney@gmail.com>
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=	Hack
+COMPONENT_VERSION= 2.020
+COMPONENT_SUMMARY= Hack, a typeface designed for source code
+COMPONENT_PROJECT_URL= http://sourcefoundry.org/hack/
+COMPONENT_FMRI=		system/font/truetype/hack
+COMPONENT_CLASSIFICATION=	System/Fonts
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=      $(COMPONENT_NAME)-$(COMPONENT_VERSION).zip
+COMPONENT_ARCHIVE_URL=	\
+	https://github.com/chrissimpkins/Hack/releases/download/v$(COMPONENT_VERSION)/$(COMPONENT_NAME)-v2_020-ttf.zip
+COMPONENT_ARCHIVE_HASH= \
+  sha256:048566ae79c580f725b68340d9d2a3b0fa125fb08c1d30cf0a7c327d07ab739a
+COMPONENT_LICENSE=	Hack Open Font License Version 2.0
+COMPONENT_LICENSE_FILE=	hack.license
+
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/justmake.mk
+include $(WS_TOP)/make-rules/ips.mk
+
+# According to other package Makefiles (like sysutils/pwgen), IPS
+# Package versions contain sequences of non-negative (for now) integers,
+# separated by dots.  Leading zeroes are not allowed, so we have to
+# transform the upstream version into a suitable version number that
+# doesn't have a leading 0 in any component.
+#
+IPS_COMPONENT_VERSION= $(shell echo $(COMPONENT_VERSION) | sed -e 's/\.0*\([1-9]\)/.\1/')
```

Comment:
Please, shorten comment to one line (or remove) and move IPS_COMPONENT_VERSION just below COMPONENT_VERSION (so that one looking at Makefile immediately could see this magic)